artifactory/commands/transferfiles/fulltransfer.go

```diff
@@ -172,9 +173,11 @@ func (m *fullTransferPhase) searchAndHandleFolderContents(params folderParams, p
 			return
 		}
 
-		// Empty folder. Add it as candidate.
+		// Add the folder as a candidate to transfer. The reason is to transfer folders with properties or empty folders.
```

Comment:
...The reason is to transfer folders... --> ...The reason is that we'd like to transfer only folders...